release-notes/release-notes.go

```diff
@@ -127,16 +133,21 @@ func main() {
 	buffer := &bytes.Buffer{}
 	for _, pr := range prs {
 		if lastVersionMerged.Before(*pr.MergedAt) && (pr.MergedAt.Before(*currentVersionMerged) || (*pr.Number == current)) {
-			// Check to see if it has the release-note label.
-			fmt.Printf(".")
-			labels, _, err := client.Issues.ListLabelsByIssue("kubernetes", "kubernetes", *pr.Number, &github.ListOptions{})
-			if err != nil {
-				fmt.Printf("Error contacting github: %v", err)
-				os.Exit(1)
-			}
-			for _, label := range labels {
-				if *label.Name == "release-note" {
-					fmt.Fprintf(buffer, "   * %s (#%d, @%s)\n", *pr.Title, *pr.Number, *pr.User.Login)
+			if !relnoteFilter {
+				fmt.Fprintf(buffer, "   * %s (#%d, @%s)\n", *pr.Title, *pr.Number, *pr.User.Login)
+			} else {
+				// Check to see if it has the release-note label.
+				fmt.Printf(".")
```

Comment:
You should print the dot ahead of the if/else branch since it would be nice to see progress in both modes.